src/python/gi/overrides/BlockDev.py

```diff
@@ -294,6 +295,13 @@ def kbd_zram_create_devices(num_devices, sizes, nstreams=None):
 __all__.append("kbd_zram_create_devices")
 
 
+_part_create_table = BlockDev.part_create_table
+@override(BlockDev.part_create_table)
+def part_create_table(disk, type, ignore_existing=True):
+    return _part_create_table(disk, type, ignore_existing)
+__all__.append("part_create_table")
+
```

Comment:
Those that can make use of python-specific features like default values for parameters or dynamic typing and conversion of the arguments. Thinking about it here, this function could probably be able to take ``type`` as a string and convert it to the BlockDev.PartTableType enum.